EntAppProject/src/main/java/com/jobook/EntAppProject/Services/JobServiceStub.java

```diff
@@ -12,48 +12,46 @@
 
 @Component
 public class JobServiceStub implements IJobService {
-	
+
 	@Autowired
 	IUserService userService;
-	
-	private List<Job> jobs; 
```

Comment:
This is a REALLY bad idea. Even though this is a stub, you have a variable scoped at a level that is far too high, and several functions directly mutate it. This could cause concurrency issues, and is generally just a bad idea, in my opinion.